### ISSUE TITLE
python_odoo script connects to all databases

### ISSUE THREAD

**Issue by sbidoul** [anybox/anybox.recipe.odoo#11]
Not sure if this is an issue or merely a question.

When using the python_odoo script generated by the recipe, it seems it connects to all available databases. This seems to be the normal Odoo behaviour, but unncessarily uses up postgres connections.

However, when using a script, we connect using session.open(db=DATABASE_NAME) and typically want to work on one database only.

Is there an easy way to make sure the script connects to only one database?

**Comment by gracinet** [comment]
@sbidoul do you still see unexpected databases in your log ? If so, any hints for reproduction ?

**Comment by sbidoul** [comment]
Yes I do. I've not had time to investigate yet, but it seems it shows database that are not completely installed. Nothing urgent in any case.

**Comment by gracinet** [comment]
ok, I'll carry on triaging / merging / fixing the other issues, then, thanks.
